internal/printer/emitcontext.go

```diff
@@ -961,3 +962,14 @@ func (c *EmitContext) GetSyntheticTrailingComments(node *ast.Node) []Synthesized
 	}
 	return nil
 }
+
+func (c *EmitContext) VisitEmbeddedStatement(node *ast.Statement, visitor *ast.NodeVisitor) *ast.Statement {
+	updated := visitor.VisitEmbeddedStatement(node)
```

Comment:
I'm sort of surprised by this, given I thought the default was to do this already...

Reply:
It looks like it doesn’t by default

https://github.com/microsoft/typescript-go/blob/52f2ee4f6e31ef7072dc4014bfc7e73649ad6a92/internal/ast/visitor.go#L74-L84

https://github.com/microsoft/typescript-go/blob/52f2ee4f6e31ef7072dc4014bfc7e73649ad6a92/internal/ast/visitor.go#L201-L209

I’m not convinced the visitor is the right place for this - WDTY? Also, the change can be implemented via a transformer-level hook or in the emitter.